examples/jenkins/README.md

```diff
@@ -9,68 +9,65 @@ the deployment works, and then tag the test version into production.
 Steps
 -----
 
-1. Follow steps 1-7 from the [sample-app](https://github.com/openshift/origin/blob/master/examples/sample-app/README.md)
+0. Unless you have built OpenShift locally, be sure  to grab the [latest oc command](https://github.com/openshift/origin/releases/latest)
 
-    At this point you should be logged in to openshift as a normal user and working with a project named `test`.
+1. Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
 
-2. Optional:  If you are going to run Jenkins somewhere other than as a deployment within your same project you will need to provide an access token to Jenkins to access your project.
+        $ oc cluster up
 
-    1. Identify the service account token secret:
+    Afterward, ensure the imagestreams are registered in the openshift namespace, as well as the
+jenkins template represented by jenkinstemplate.json by running these commands as a cluster admin:
```

Comment:
copy/paste fail ... i'm just going to remove "represented by jenksinstemplate.json" and say "as well as the jenkins templates by running ..."